axlearn/common/launch_trainer_main.py

```diff
@@ -13,7 +13,6 @@ def main(_):
     launch.setup()
     trainer_config = launch_trainer.get_trainer_config()
     trainer_config.set(recorder=config_for_function(lambda: measurement.global_recorder))
-    measurement.start_monitoring()
```

Comment:
This will also be another breaking API change cc @Ethanlm

Reply:
As @heyaudace mentioned, this also breaks the current API. Can we address it too?

Reply:
Sure, added it back. Other RecorderTypes implementing this should work as is.